com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -171,12 +222,12 @@ float3 ReconstructViewPos(float2 uv, float depth)
 // High:   5 taps on each direction: | z | x | * | y | w |
 // https://atyuwen.github.io/posts/normal-reconstruction/
 // https://wickedengine.net/2019/09/22/improved-normal-reconstruction-from-depth/
-float3 ReconstructNormal(float2 uv, float depth, float3 vpos)
```

Comment:
This seems generic, maybe put it in Universal utility functions.

Reply:
That's the plan. It will be done by @lukaschod as he's doing similar things with the Decal work.
He's also going to look if some improvements can be made.